### PR TITLE
Updating apirequestcount admin ack with a better query and to show an…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…other user to ignore

Previews: 
* https://deploy-preview-40561--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-prepare.html#update-preparing-evaluate-apirequestcount_updating-cluster-prepare
* https://deploy-preview-40561--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-prepare.html#update-preparing-evaluate-apirequestcount-workloads_updating-cluster-prepare

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@sanchezl @wangke19 Can you please review? 

Preview: https://deploy-preview-40561--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-prepare.html#update-preparing-evaluate-apirequestcount_updating-cluster-prepare

**Review by sanchezl — CHANGES_REQUESTED** [review]

File: modules/update-preparing-evaluate-apirequestcount.adoc [hunk 1]
```diff
@@ -36,8 +36,8 @@ installplans.v1alpha1.operators.coreos.com                     93
 ====
 You can safely ignore the following entries that appear in the results:
 
-* `system:serviceaccount:kube-system:generic-garbage-collector` appears in the results because it walks through all registered APIs searching for resources to remove.
-* `system:kube-controller-manager` appears in the results because it walks through all resources to count them while enforcing quotas.
+* The `system:serviceaccount:kube-system:generic-garbage-collector` user might appear in the results because it walks through all registered APIs searching for resources to remove.
+* The `system:kube-controller-manager` and `system:cluster-policy-controller` users might appear in the results because they walk through all resources to count them while enforcing various policies.
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply modules/update-preparing-evaluate-apirequestcount.adoc]: Updated!

**Review by sanchezl — APPROVED** [review]
/lgtm

**Comment by bergerhoffer** [comment]
@wangke19 Can you please QE review this?

Preview: https://deploy-preview-40561--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-prepare.html#update-preparing-evaluate-apirequestcount_updating-cluster-prepare

**Pull request merged**